service/create/security_group.go

```diff
@@ -26,14 +26,18 @@ type rulesInput struct {
 }
 
 const (
+	allowAllPorts        = -1
 	calicoBGPNetworkPort = 179
+	httpPort             = 80
+	httpsPort            = 443
```

Comment:
This naming also should be consistent. See my other comment about `allow` prefix.